evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -149,13 +149,13 @@
     {% endif %}
 {% elif info_only or not request.user.is_manager %}
     <td style="width: 20%">
-        <span data-toggle="tooltip" data-placement="top" title="{% trans 'Number of participants' %}">
+        <span {% if evaluation.num_participants == 0 %}class="text-danger" {% endif %}data-toggle="tooltip" data-placement="top" title="{% trans 'Number of participants' %}">
```

Comment:
(same performance consideration might apply here and below)

Reply:
`num_participants` is already annotated